src/hotspot/share/runtime/objectMonitor.hpp

```diff
@@ -384,6 +384,8 @@ class ObjectMonitor : public CHeapObj<mtObjectMonitor> {
   void      wait(jlong millis, bool interruptible, TRAPS);
   void      notify(TRAPS);
   void      notifyAll(TRAPS);
+  void      quick_notify(TRAPS);
+  void      quick_notifyAll(TRAPS);
```

Comment:
These should not be declared with TRAPS as they don't throw exceptions (and must not!). These should simply take a `JavaThread* current` parameter.